test/recipes/30-test_evp_data/evppkey.txt

```diff
@@ -378,7 +378,7 @@ Reason = operation not supported for this keytype
 
 Sign = RSA-2048
 Ctrl = rsa_mgf1_md:sha1
-Result = PKEY_CTRL_INVALID
+Result = PKEY_CTRL_ERROR
```

Comment:
Why do we get this change in behaviour?

Reply:
The commit message explains it (d92bb0cdda3a40f4dc445090f04b8fed5749230b):

## test/recipes/30-test_evp_data/evppkey.txt

Tests that go through provider cannot recognise PKEY_CTRL_INVALID from
PKEY_CTRL_ERROR any more, because provided implementations' param
setting functions return 0 or 1.